index.html

```diff
@@ -0,0 +1,952 @@
+                                    world."</p>
+
+                            </div>
+                        </article>
+
+                        <article class="card">
+                            <img src="https://d.newsweek.com/en/full/1514134/ancient-egypt-jewelry.png?w=397&h=265&f=82f483bef40556f0113fec07d2a5315b"
+                                class="card-img-top" alt="Newsweek Momentum">
+                            <div class="card-body">
+                                <h5 class="card-title">The Newsweek Momentum Awards</h5>
+                                <p class="card-text">The Newsweek Momentum Awards will recognize five individuals around
+                                    the
+                                    globe, as well as one city, which will be chosen as the "smartest city in the
+                                    world."</p>
+
+                            </div>
+                        </article>
+                    </div>
+                    <div class="col">
+                        <h6 class="hidden">Placeholder</h6>
+                        <article class="card">
+                            <img src="https://d.newsweek.com/en/full/1515608/meteor-shooting-star.jpg?w=397&h=265&f=1f4f4b18d87b3236fdc25b0f5e94054d"
+                                class="card-img-top" alt="Newsweek Momentum">
+                            <div class="card-body">
+                                <h5 class="card-title">The Newsweek Momentum Awards</h5>
+                                <p class="card-text">The Newsweek Momentum Awards will recognize five individuals around
+                                    the
+                                    globe, as well as one city, which will be chosen as the "smartest city in the
+                                    world."</p>
+
+                            </div>
+                        </article>
+                        <article class="card">
+                            <img src="https://d.newsweek.com/en/full/1513906/cul-books-01-748319631.jpg?w=397&h=265&f=a76b1f70348747e4d543c615e152e7f7"
+                                class="card-img-top" alt="Newsweek Momentum">
+                            <div class="card-body">
+                                <h5 class="card-title">The Newsweek Momentum Awards</h5>
+                                <p class="card-text">The Newsweek Momentum Awards will recognize five individuals around
+                                    the
+                                    globe, as well as one city, which will be chosen as the "smartest city in the
+                                    world."</p>
+
+                            </div>
+                        </article>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col">
+                <h6 class="more-stories-title">MORE STORIES</h6>
```

Comment:
### `HINT`

Consider styling this and all other occurrences. (For example, giving it font color and weight as in the source page). See below:

| Current | Preferred |
| :-----: | :-----: |
| ![more stories current](https://user-images.githubusercontent.com/36057474/62750140-e5fc9400-ba56-11e9-961b-d508e5f79409.png) | ![more stories preffered](https://user-images.githubusercontent.com/36057474/62750149-edbc3880-ba56-11e9-9e0e-d3943c93ffa1.png) |

#### Do this for all other occurrences.